pallets/sequencer-staking/src/lib.rs

```diff
@@ -492,20 +492,11 @@ impl<T: Config> Pallet<T> {
 	// 	Self::maybe_remove_sequencers_from_active_set(exiting_collators);
 	// }
 
-	fn maybe_remove_sequencers_from_active_set(exiting_collators: BTreeSet<T::AccountId>) {
-		let to_remove = ActiveSequencers::<T>::get().into_iter().map(|(chain, active)| {
-			(
-				chain,
-				active
-					.iter()
-					.filter(|seq| exiting_collators.contains(seq))
-					.cloned()
-					.collect::<BTreeSet<_>>(),
-			)
-		});
-
-		for (chain, sequencers_set) in to_remove {
-			Self::remove_sequencers_from_active_set(chain, sequencers_set);
+	fn maybe_remove_sequencer_from_active_set(chain: T::ChainId, sequencer: T::AccountId) {
+		if <SequencerStake<T>>::get((sequencer.clone(), chain)) < MinimalStakeAmount::<T>::get() &&
```

Comment:
The slashed sequencer now won't leave the active sequencer set. I think that was a requirement. Becoz if something is wrong with the guy's setup he will keep getting slashed this he has very little money left...